en,zh: add 1.4.0-beta.1 release notes (#2050)
pingcap/docs-tidb-operator: en/releases/release-1.4.0-beta.1.md

---
title: TiDB Operator 1.4.0-beta.1 Release Notes
---

# TiDB Operator 1.4.0-beta.1 Release Notes

Release date: October 27, 2022

TiDB Operator version: 1.4.0-beta.1

## New Feature

> - TiDB Operator supports AWS EBS snapshot Backup and Restore ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky)). the feature has the advantages:

Reviewer: ```suggestion
- TiDB Operator supports AWS EBS snapshot Backup and Restore ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky)). This feature has the following advantages:
```

> - TiDB Operator supports AWS EBS snapshot Backup and Restore ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky)). the feature has the advantages:

Reviewer 2: ```suggestion
- TiDB Operator supports snapshot backup and restore based on Amazon EBS ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky)). This feature has the following benefits:

```

>     - Reduce backup impact (qps < 5%)

Reviewer 2: ```suggestion
    - Reduce backup impact (QPS < 5%)
```

>     - Reduce backup impact (qps < 5%)

Reviewer: ```suggestion
    - Reduce backup impact (QPS < 5%)
```

---
title: TiDB Operator 1.4.0-beta.1 Release Notes
---

# TiDB Operator 1.4.0-beta.1 Release Notes

Release date: October 27, 2022

TiDB Operator version: 1.4.0-beta.1

## New Feature

- TiDB Operator supports snapshot backup and restore based on Amazon EBS ([#4698](https://github.com/pingcap/tidb-operator/pull/4698), [@gozssky](https://github.com/gozssky)). This feature has the following benefits:

>     - Reduce backup impact (QPS < 3%)

Reviewer: ```suggestion
    - Reduce backup impact (QPS < 5%)
```

>     - Reduce backup impact (QPS < 5%)

Reviewer 2: ```suggestion
    - Reduce the impact of backup on QPS to less than 5%
```
